test/unit/sql/where.test.js

```diff
@@ -908,6 +908,24 @@ suite(Support.getTestDialectTeaser('SQL'), () => {
       });
     }
 
+    if (current.dialect.supports.REGEXP) {
+      suite('$regexp', () => {
+        testsql('username', {
+          $regexp: '^sw.*r$'
```

Comment:
Could you add a test for using a regexp literal? We might have to call `toString()` somewhere to make it work, but I think it would be slick because you wouldn't have to double-escape backslashes. I would also change the docs then